pages/mint-nft/index.vue

```diff
@@ -585,8 +601,12 @@ async function onISCNIDInput () {
       iscnOwner.value = owner
       step.value = 2
     } else if (iscnIdInput.value.startsWith('likenft')) {
+      isRestockingNFT.value = true
```

Comment:
I think we can fetch `${LIKE_CO_API}/likernft/book/store/${iscnIdInput.value}` once to see if existing listing exists
Set `isRestockingNFT` to true only if it exists/not 404

Reply:
[🎨 Enhance existing listing check](https://github.com/likecoin/nft-book-press/pull/330/commits/45107d8c515c1b3914e0a1fb4df2676b62e443ab)